typescript/packages/dash-components/src/WellCompletions/components/Settings/SortTable.tsx

```diff
@@ -6,13 +6,11 @@ import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { deleteSortKey, updateSortKey } from "../../redux/actions";
 import { WellCompletionsState } from "../../redux/store";
-import { SortDirection } from "../../redux/types";
 import {
-    SORT_BY_COMPLETION_DATE,
-    SORT_BY_NAME,
-    // eslint-disable-next-line prettier/prettier
-    SORT_BY_STRATIGRAPHY_DEPTH
-} from "../../utils/sort";
+    SortBy,
+    SortByString,
```

Comment:
`SortByString` sounds like something to use for sorting by a given string. Maybe `SortByEnumToStringMapping`?